powersimdata/input/transform_profile.py

```diff
@@ -125,6 +135,6 @@ def get_profile(self, name):
         if name not in possible:
             raise ValueError("Choose from %s" % " | ".join(possible))
         elif name == "demand":
-            return self._get_demand_profile()
+            return self._slice(self._get_demand_profile())
         else:
-            return self._get_renewable_profile(name)
+            return self._slice(self._get_renewable_profile(name))
```

Comment:
I think we use this to write the profiles that are uploaded to the server, which I assume are used by REISE.jl directly. Is that fine?

Reply:
Oh yeah, good catch. We don't want to change the files that are prepared for REISE.jl, at least not right now.

Reply:
Perhaps a bool parameter that defaults to this behavior, but in `SimulationInput.prepare_profile` we pass the value that says to preserve the full profile?

Reply:
I guess in develop branch that would be in input/export_data.py (I have a different branch checked out where it's called directly)

Reply:
Added in the latest commit. `SimulationInput.prepare_profile` now has a `slice` flag (defaulting to False) which gets passed to `input.export_data.export_transformed_profile` (default is True), which gets passed to `TransformProfile` (default is True).

When we refactor REISE.jl, I think we can just flip the `slice` flag in the `SimulationInput.prepare_profile` call, and everything else should work as expected.